v3/v3_structs.go

```diff
@@ -2572,3 +2572,36 @@ type AddressGroupListResponse struct {
 	Metadata *ListMetadataOutput      `json:"metadata,omitempty"`
 	Entities []*AddressGroupListEntry `json:"entities,omitempty"`
 }
+
+type GroupsRequestedAttribute struct {
+	Attribute *string `json:"attribute"`
+}
```

Comment:
consider adding a newline between type declarations.

Reply:
I'll fix it in the next patch.